artemis/src/main/java/tech/pegasys/artemis/GenesisCommand.java

```diff
@@ -0,0 +1,95 @@
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis;
+
+import static tech.pegasys.artemis.datastructures.util.SimpleOffsetSerializer.serialize;
+import static tech.pegasys.artemis.util.alogger.ALogger.STDOUT;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.List;
+import org.apache.logging.log4j.Level;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Mixin;
+import picocli.CommandLine.Option;
+import tech.pegasys.artemis.datastructures.state.BeaconState;
+import tech.pegasys.artemis.datastructures.util.MockStartValidatorKeyPairFactory;
+import tech.pegasys.artemis.statetransition.util.StartupUtil;
+import tech.pegasys.artemis.util.bls.BLSKeyPair;
+import tech.pegasys.artemis.util.cli.VersionProvider;
+
+@Command(
+    name = "genesis",
+    description = "Commands for generating genesis state",
+    abbreviateSynopsis = true,
+    mixinStandardHelpOptions = true,
+    versionProvider = VersionProvider.class,
+    synopsisHeading = "%n",
+    descriptionHeading = "%nDescription:%n%n",
+    optionListHeading = "%nOptions:%n",
+    footerHeading = "%n",
+    footer = "Artemis is licensed under the Apache License 2.0")
+public class GenesisCommand {
+
+  @Command(
+      name = "mock",
+      description = "Generate a mock genesis state",
+      mixinStandardHelpOptions = true,
+      abbreviateSynopsis = true,
+      versionProvider = VersionProvider.class,
+      synopsisHeading = "%n",
+      descriptionHeading = "%nDescription:%n%n",
+      optionListHeading = "%nOptions:%n",
+      footerHeading = "%n",
+      footer = "Artemis is licensed under the Apache License 2.0")
+  public void generate(@Mixin MockGenesisParams params) throws IOException {
+    FileOutputStream fileStream = new FileOutputStream(params.outputFile);
+
+    STDOUT.log(
+        Level.INFO,
+        String.format(
+            "Generating mock genesis state for %d validators at genesis time %d",
+            params.validatorCount, params.genesisTime));
+
+    final long genesisTime = params.genesisTime;
+    final List<BLSKeyPair> validatorKeys =
+        new MockStartValidatorKeyPairFactory().generateKeyPairs(0, params.validatorCount);
+    final BeaconState genesisState =
+        StartupUtil.createMockedStartInitialBeaconState(genesisTime, validatorKeys);
+
+    STDOUT.log(Level.INFO, String.format("Saving genesis state to file: %s", params.outputFile));
+    fileStream.write(serialize(genesisState).toArrayUnsafe());
+    fileStream.close();
+  }
+
+  public static class MockGenesisParams {
+    @Option(
+        names = {"-o", "--outputFile"},
+        paramLabel = "<FILENAME>",
+        description = "Path/filename of the output file")
+    private String outputFile = "./mock-genesis-state.bin";
```

Comment:
When running via docker (especially in acceptance tests) it would be really useful if this wrote the genesis state to stdout if no output file is specified. Otherwise the result gets written to a file inside the docker container and you have to jump through hoops to get it back out.